kubernetes/pro/deployInfrastructure.ps1

```diff
@@ -0,0 +1,57 @@
+$namespace = "featbit"
+
+
+# Check if namespace exists
+if (-not (kubectl get namespaces $namespace -o name)) {
+    Write-Host "Namespace '$namespace' does not exist. Creating..."
+
+    # Create namespace
+    kubectl create namespace $namespace
+
+    Write-Host "Namespace '$namespace' created."
+} else {
+    Write-Host "Namespace '$namespace' already exists."
+}
+
+kubectl config set-context --current --namespace=$namespace
+
+kubectl apply -f .\infrastructure\mongodb-init-configMap.yaml
+
+kubectl apply -f .\infrastructure\mongodb-persistentvolumeclaim.yaml 
+
+kubectl apply -f .\infrastructure\mongodb-service.yaml
+
+kubectl apply -f .\infrastructure\mongodb-pod.yaml  
+
+kubectl apply -f .\infrastructure\redis-persistentvolumeclaim.yaml
+
+kubectl apply -f .\infrastructure\redis-service.yaml
+
+kubectl apply -f .\infrastructure\redis-pod.yaml
+
+kubectl apply -f .\infrastructure\zookeeper-persistentvolumeclaim.yaml
+
+kubectl apply -f .\infrastructure\zookeeper-service.yaml
+
+kubectl apply -f .\infrastructure\zookeeper-pod.yaml
+
+kubectl apply -f .\infrastructure\clickhouse-agent-config-configMap.yaml
+
+kubectl apply -f .\infrastructure\clickhouse-configmap-etc.yaml
+
+kubectl apply -f .\infrastructure\clickhouse-configmap-etc.yaml
+
+kubectl apply -f .\infrastructure\clickhouse-server-persistentvolumeclaim.yaml
+
+kubectl apply -f .\infrastructure\clickhouse-server-service.yaml
+
+kubectl apply -f .\infrastructure\clickhouse-server-pod.yaml
+
+kubectl apply -f .\infrastructure\kafka-persistentvolumeclaim.yaml
+
+kubectl apply -f .\infrastructure\kafka-service.yaml
+
+kubectl apply -f .\infrastructure\kafka-pod.yaml
+
+# kubectl apply -f .\infrastructure\init-kafka-topics-deployment.yaml
```

Comment:
Are you trying to use this to bootstrap kafka topics? You can include this file if it works or remove it if not

Reply:
I was trying another way to do it an forgot to remove the commented line.  It's removed with commit e2c027d